lib/sinon.js

```diff
@@ -14,6 +14,8 @@ module.exports = exports = require("./sinon/util/core");
 
 exports.extend = require("./sinon/extend");
 exports.match = match;
+exports.spy = require("./sinon/spy");
+exports.spyCall = require("./sinon/call");
```

Comment:
I think this was used internally. I think it can be removed since it would be released with a major. @cjohansen do you recall?